### PR TITLE
CSS - Canvas - Medical - Addy Abid

### PR DESCRIPTION
**Pull request by AddyAbid**
https://user-images.githubusercontent.com/85080021/130555128-2a731ce1-133b-473f-828b-4eb9e6c9e943.mp4

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Addy! Your alignment of the button and text in the "Ready to learn more?" row is a bit off as compared to the example, please revise and resubmit with a new screenshot to show the fix.

**Comment by AddyAbid** [comment]
_Users_addyabid_repos_c0821-code-solutions_css-canvas-medical_index html

**Comment by AddyAbid** [comment]
_Users_addyabid_repos_c0821-code-solutions_css-canvas-medical_index html (1)

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      </div>
+      <div class="nav">
+        <a href="./additional-examples/about-us.png">About Us</a>
+      </div>
+      <div class="nav">
+        <a href="./additional-examples/blog.png">Blog</a>
+      </div>
+      <div class="column-15">
+        <button class="contact-us" type="button">Contact Us</button>
+      </div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:32]: Shouldn't all of these be a child of a `div` with the `column-50` class? I doubt you want just your first link to take up 50% of the available width on its own.

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+.emr-text {
+  font-family: 'Tiempos Headline', sans-serif, serif;
+}
+
+.primary {
+  position: absolute;
+  top: 40px;
+  color: rgb(255, 255, 255);
+  font-family: 'Tiempos Headline', sans-serif, serif;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:74]: Why does this need to be absolute? `col-text` is already `absolute`. If you need more space between this element and its siblings, use `margin` or `padding`

File: css-canvas-medical/styles.css [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+  bottom: 155px;
+  margin-left: 21px;
+}
+
+.remember {
+  position: absolute;
+  top: 100px;
+  color: rgb(233, 232, 230);
+  line-height: 25px;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:90]: Same here, this likely doesn't need to be `absolute`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great updates on this! A few more notes for you:

**Comment by AddyAbid** [comment]
_Users_addyabid_repos_c0821-code-solutions_css-canvas-medical_index html (2)

**Review by codingcodymiller — APPROVED** [review]
Great work!!!

**Pull request merged**